org-formation/720-client-vpn/vpn.njk

```diff
@@ -102,6 +102,26 @@ Resources:
       SubnetId: {{ subnet_id }}
 {% endfor %}
 
+{# Authorization and route between VPN and public internet #}
+  EndpointAuthorizationPublic:
+    Type: AWS::EC2::ClientVpnAuthorizationRule
+    Properties:
+      Description: "authorization between VPN and public internet"
+      AccessGroupId: {{ access_group }}
```

Comment:
Will `{{ access_group }}` resolve outside of the loop on ln 127?

Reply:
ah yes, nice catch.  This will not resolve here.  We are gonna need a loop to give all access groups access to 0.0.0.0/0

Reply:
fixed